### PR TITLE
PHPUnit 8 update - fixing tests

### PR DESCRIPTION
**Pull request by dregad**
This is in preparation for when splitbrain/dokuwiki#3404 gets merged.

### CODE REVIEW

File: helper/plan.php [hunk 1]
```diff
@@ -623,7 +623,7 @@ protected function stepThroughNamespaces() {
         $lines = explode("\n", $lines);
 
         foreach($lines as $line) {
-            list($src, $dst) = explode("\t", trim($line));
+            list($src, $dst) = explode("\t", trim($line) . "\t");
```
Comment (michitux) [inline helper/plan.php:626]: Why is this necessary? Is this because the root namespace is an empty string that is removed by trim? If this is the case, could we maybe not trim the tab character?

**Review by michitux — CHANGES_REQUESTED** [review]
This breaks the continuous integration runs with PHP versions before 7.1. I think it is okay to drop them and include some more recent PHP versions instead. Maybe we can also move old-stable tests to more recent PHP versions - as this plug-in is quite popular, I want to ensure that both currently supported DokuWiki versions are supported by the move plug-in. We should also move to GitHub actions, but this can be done later. dokufreaks/dokuwiki-plugin-wizard#8 provides a template for the move to GitHub actions.

File: helper/plan.php (hunk 1 above)
Comment (dregad) [reply helper/plan.php:626]: Without this, the unit tests fail on PHP 8  with an undefined array key error (6 occurrences):
```
1) plugin_move_namespace_move_test::test_move_wiki_namespace
Undefined array key 1

/home/dregad/dev/dokuwiki/move/helper/plan.php:626
/home/dregad/dev/dokuwiki/move/helper/plan.php:395
/home/dregad/dev/dokuwiki/move/_test/namespace_move.test.php:61
```

The `list($src, $dst)` assignment expects an array with at least 2 items; if there is only one (i.e. if *$line* is a string without a tab character, then the error is triggered.

To be honest, I just fixed the symptoms without fully analyzing the root cause, so maybe this is not appropriate. 

Actually looking into this, the data read from the *namespaces* file into the $lines array has a trailing newline `wiki\tfoo:\n` so the explode() statement at line 623 creates an array with 2 items; the first line  is parsed correctly, but the second is an empty string that triggers the error.

So maybe the right fix would be to rtrim $lines before calling explode so this empty line is not processed.


 I just fixed the symptoms
, caused by
